docs/user-guide/walkthrough/index.md

```diff
@@ -64,11 +53,12 @@ $ kubectl get pods
 
 On most providers, the pod IPs are not externally accessible. The easiest way to test that the pod is working is to create a busybox pod and exec commands on it remotely. See the [command execution documentation](/docs/user-guide/getting-into-containers/) for details.
 
-Provided the pod IP is accessible, you should be able to access its http endpoint with curl on port 80:
+Provided the pod IP is accessible, you should be able to access its http endpoint with wget on port 80:
```

Comment:
Oh, gotcha! Yeah, I think that syntax looks fine. The shell usually gives you somesuch preamble.